<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>About</class>
 <widget class="QDialog" name="About">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>About</string>
  </property>
  <widget class="QLabel" name="lblAboutLogo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>123</width>
     <height>335</height>
    </rect>
   </property>
   <property name="text">
    <string>*Pixmap*</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblAboutText">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>54</y>
     <width>371</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string>*Text*</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="lblAboutTitle">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>22</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>*Title*</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblUrl">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>360</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>*URL*</string>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOk">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>398</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
